src/app/curriculum-courses/chapter-details/chapter-details.page.ts

```diff
@@ -472,7 +478,12 @@ export class ChapterDetailsPage implements OnInit, OnDestroy {
         this.objRollup,
         // this.corRelationList
       );
-      this.navigateToChildrenDetailsPage(this.nextContent, 1);
+
+      const telemetryDetails = {
+        pageId: PageId.COURSE_DETAIL,
```

Comment:
PageId is incorrect.

Reply:
Was a mistake, will change to "CHAPTER_DETAILS"